api/sheets/get_variable.md

```diff
@@ -1,111 +1,119 @@
 ---
 layout: default
-title: Statistical Variable Value
+title: Returning Statistical Variables
 nav_order: 3
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the value of a statistical variable at a given place and time
+# Returning Statistical Variables
```

Comment:
could we revert to the previous version of this. we are not returning the variables, but rather the values associated with them.

Reply:
Sure, I changed it to "Getting the Value of a Statistical Variable at a Given Place and Time". Do you also want to change the title of the page to this wording?